app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user
+  
+  def create
+    @video = Video.find(params[:id])
+    @review = @video.reviews.build(params.require(:review).permit(:rating, :body))
+    @review.user = current_user
+    if @review.save
+      flash[:notice] = 'Your comment was added'
+      redirect_to @video
+    else
+      flash[:error] = 'Your comment was not added'
```

Comment:
You should use `flash.now` when you `render` otherwise the message pops on next request as well.